src/GraphQL.NewtonsoftJson/ExecutionResultJsonConverter.cs

```diff
@@ -11,13 +12,23 @@ namespace GraphQL.NewtonsoftJson
     public class ExecutionResultJsonConverter : JsonConverter
     {
         private readonly IErrorInfoProvider _errorInfoProvider;
+        private readonly NamingStrategy _namingStrategy;
 
         /// <summary>
         /// Initializes a new instance with the specified <see cref="IErrorInfoProvider"/>.
         /// </summary>
         public ExecutionResultJsonConverter(IErrorInfoProvider errorInfoProvider)
+            : this(errorInfoProvider, null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance with the specified <see cref="IErrorInfoProvider"/> and <see cref="NamingStrategy"/>.
+        /// </summary>
+        public ExecutionResultJsonConverter(IErrorInfoProvider errorInfoProvider, NamingStrategy namingStrategy)
```

Comment:
```suggestion
        public ExecutionResultJsonConverter(IErrorInfoProvider errorInfoProvider, NamingStrategy? namingStrategy)
```
`DefaultContractResolver.NamingStrategy` is nullable.

Reply:
Yes, but this project does not have NRT annotations enabled.  We need to finish that up for all the other projects in the solution.  Only the main project has NRT enabled.

Reply:
Please add //TODO comment here

Reply:
Todo what?!? This NRT annotation is obvious and won’t compile if incorrect once annotations are enabled. Further, many changes will need to be made.

Reply:
ok